<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>paramsLLE</class>
 <widget class="QWidget" name="paramsLLE">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="dimCountLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Projected Dim</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="dimCountSpin">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>60</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="knnLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>k - NN</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="knnSpin">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>60</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>200</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="eigenGraph">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>K-PCA Eigenvalues</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="eigenList">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
